applications/Crosschainstorage.md

```diff
@@ -0,0 +1,138 @@
+# Crust Grant Proposal
+
+* Project Name: CCS - Cross Chain Storage Platform
+* Team Name: Zhu, Wenfeng (朱文峰); Gao, Le (高乐); Wang, Ruijun (王瑞俊); Sang, Junjun (桑俊俊); Dong, Yuansheng (董远胜); Liu, Qi (刘琦)
+* Payment Address: 0x17389d755cb720e5e7e12d0fb926c0d53a2846d6 (USDT)
+* Receivable Token: USDT – ERC20
+
+
+### Overview
+
+* A project name: CCS - Cross Chain Storage Platform
+* A brief description of your project: In the ear of data explosion and related storage request, we want to create a decentralized, highly motivated storage platform. We will provide customer (individual and business) with a more secure, privacy-protected and cost-effective data storage service. 
+* CCS will build multiple connection adapter incentive layers; Crust will be the pilot and main storage channel layer for CCS.
+* Our team sees an opportunity to consolidate multiple decentralized storage providers to build a platform like “去哪儿” in air-tickets booking network, which provide business clients with most optimized data / files storage solutions.
```

Comment:
What is the difference between this project and Cold Stack

Reply:
1, For users, there is a better UI experience, which is more in line with current customers' usage habits, shields many blockchain usage thresholds (such as wallets, etc.), and lowers migration costs. Coldstack currently directly connects to the neutralization layer in one step, exposing a lot of complexity to customers and increasing the cost of use.

2, On the customer side: CCS currently focuses on the needs of corporate customers as the first step, then in the future will  develops individual users version. At the same time, engage domestic market first and then globalwide. Coldstack pays more attention to individual users, most of them are foreign customers.

3, Crust integration: coldstack has not yet done crust integration, and is still planning. CCS has taken the lead in completing the MVP verification of crust integration. Crust is the first choice.

Reply:
I have dropped an email to you, could we have a meeting?